### PR TITLE
Appx.B Suggested Reading - v4 Migration Cleanup - Fixes #91

### LINKED ISSUE #91: Appx.B Suggested Reading - v4 Migration Cleanup
- [ ] "wikilinks" during migration the OWASP Mediawiki links were tagged "wikilink" these needs to be converted to proper absolute URLs.
  - Unless the wikilink is to content within the OTG, in which case it should be the proper new github relative URL (URL Encoded).
- [ ] Foot notes/references should use standard in-line links. (One of the first two here: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links)
    - If a foot note is really required use numeric reference type tags, ex: Content ref: `[[1]]` Footer ref:  `[1]: http://example.org`
- [ ] Images need to be saved to <chapter (or sub-section)>/images/<image.ext> and linked relatively like images/image.ext. (Images in a folder along side the relevant content. [Even if that means duplicates in the overall project.])
- [ ] Any spurious `/` terminating lines, etc. need to be removed and appropriately fixed up for markdown (it's likely a spot that requires a double line-feed prior to an image or other element).
- [ ] In some cases (due to the source of the text) there are broken apostrophes `'` they should just be restored.
- [ ] HTML entities need to be properly handled.
  - Occurrences of `\$` should become `&dollar;`
  - At symbols `@` should become `&commat;`
- [ ] Headers should be properly formatted with leading # (hashtags) per: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers

### PR DESCRIPTION
**Pull request by rejahrehim**
This PR covers issue #<issue number>.

- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Fix links
- Fix linting errors

### CODE REVIEW

File: document/Appx.B_Suggested_Reading/Appx.B_Suggested_Reading.md [hunk 1]
```diff
@@ -1,319 +1,138 @@
...
+- [Exploiting Software: How to Break Code, by Gary McGraw and Greg Hoglund, published by Addison-Wesley Pub Co, ISBN 0201786958 (2004)](http://www.exploitingsoftware.com)
 
-<!-- -->
+- [The Hacker's Handbook: The Strategy behind Breaking into and Defending Networks, By Susan Young, Dave Aitel, Auerbach Publications, ISBN: 0849308887 (2005)](http://books.google.com/books?id=AO2fsAPVC34C&printsec=frontcover&source=gbs_ge_summary_r&cad=0#v=onepage&q&f=false)
 
--   Software Security Testing: Software Assurance Pocket guide Series: Development, Volume III - <https://buildsecurityin.us-cert.gov/swa/downloads/SoftwareSecurityTesting_PocketGuide_1%200_05182012_PostOnline.pdf>
+- [Hacking Exposed: Web Applications 3, by Joel Scambray, Vinvent Liu, Caleb Sima, published by McGraw-Hill Osborne Media, ISBN 007222438X (2010)](http://www.webhackingexposed.com/)
 
-<!-- -->
+- The Web Application Hacker's Handbook: Finding and Exploiting Security Flaws, 2nd Edition - [published by Dafydd Stuttard, Marcus Pinto, ISBN 9781118026472 (2011)
```
Comment (ThunderSon) [inline document/Appx.B_Suggested_Reading/Appx.B_Suggested_Reading.md]: Missing ending bracket? `]`

File: document/Appx.B_Suggested_Reading/Appx.B_Suggested_Reading.md [hunk 2]
```diff
@@ -1,319 +1,138 @@
...
+- [Innocent Code: A Security Wake-Up Call for Web Programmers, by Sverre Huseby, published by John Wiley & Sons, ISBN 0470857447(2004)](http://innocentcode.thathost.com)
+  - [Online version available here](http://books.google.com/books?id=RjVjgPQsKogC&printsec=frontcover&source=gbs_ge_summary_r&cad=0#v=onepage&q&f=false)
 
-\]\]
+- Mastering the Requirements Process, by Suzanne Robertson and James Robertson, published by Addison-Wesley Professional, ISBN 0201360462
+  - [Online version available here](http://books.google.com/books?id=SN4WegDHVCcC&printsec=frontcover&source=gbs_ge_summary_r&cad=0#v=onepage&q&f=false)
 
-Books
------
+- [Secure Coding: Principles and Practices, by Mark Graff and Kenneth R. Van Wyk, published by O’Reilly, ISBN 0596002424 (2003)](http://www.securecoding.org)
```
Comment (ThunderSon) [inline document/Appx.B_Suggested_Reading/Appx.B_Suggested_Reading.md]: Dead link

File: document/Appx.B_Suggested_Reading/Appx.B_Suggested_Reading.md [hunk 3]
```diff
@@ -1,319 +1,138 @@
...
+- Writing Secure Code, by Mike Howard and David LeBlanc, published by Microsoft Press, ISBN 0735617228 (2004)
 
--   The Hacker's Handbook: The Strategy behind Breaking into and Defending Networks, By Susan Young, Dave Aitel, Auerbach Publications, ISBN: 0849308887 (2005)
+## Useful Websites
 
-<!-- -->
+- [Build Security In](https://buildsecurityin.us-cert.gov/bsi/home.html)
 
--   + Online version available at: <http://books.google.com/books?id=AO2fsAPVC34C&printsec=frontcover&source=gbs_ge_summary_r&cad=0#v=onepage&q&f=false>
+- [Build Security In – Security-Specific Bibliography](https://buildsecurityin.us-cert.gov/bsi/articles/best-practices/measurement/1070-BSI.html)
```
Comment (ThunderSon) [inline document/Appx.B_Suggested_Reading/Appx.B_Suggested_Reading.md]: Duplicate link as BSI is now at the same spot. Duplicate to the link above it

File: document/Appx.B_Suggested_Reading/Appx.B_Suggested_Reading.md [hunk 4]
```diff
@@ -1,319 +1,138 @@
...
 
-<!-- -->
+- [CERT Secure Coding](http://www.cert.org/secure-coding/)
 
--   Hacking Exposed: Web Applications 3, by Joel Scambray, Vinvent Liu, Caleb Sima, published by McGraw-Hill Osborne Media, ISBN 007222438X (2010) - <http://www.webhackingexposed.com/>
+- [CERT Secure Coding Standards](https://www.securecoding.cert.org/confluence/display/seccode/CERT+Secure+Coding+Standards)
```
Comment (ThunderSon) [inline document/Appx.B_Suggested_Reading/Appx.B_Suggested_Reading.md]: Page not found

File: document/Appx.B_Suggested_Reading/Appx.B_Suggested_Reading.md [hunk 5]
```diff
@@ -1,319 +1,138 @@
...
 
-<!-- -->
+- [Exploit and Vulnerability Databases](https://buildsecurityin.us-cert.gov/swa/database.html)
```
Comment (ThunderSon) [inline document/Appx.B_Suggested_Reading/Appx.B_Suggested_Reading.md]: Duplicate BSI link to the first link in this section

Comment (rejahrehim) [reply document/Appx.B_Suggested_Reading/Appx.B_Suggested_Reading.md]: Updated

**Pull request merged**